fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -448,6 +449,7 @@ ansiNonReserved
     | INPUTFORMAT
     | INSERT
     | INTERVAL
+    | ISNULL
```

Comment:
should put it into nonReserved list too

Reply:
http://www.postgres.cn/docs/9.3/sql-keywords-appendix.html
ISNULL is not a keyword in SQL standard.